lib/nav/page-list.vue

```diff
@@ -320,29 +321,35 @@
         return postJSON(prefix + searchRoute, query).then((res) => {
           const hits = _.get(res, 'hits.hits') || [],
             total = _.get(res, 'hits.total'),
-            pages = _.map(hits, hit => Object.assign({}, hit._source, { uri: hit._id }));
+            pages = _.map(hits, (hit) => Object.assign({}, hit._source, { uri: hit._id })),
```

Comment:
We can use object destructuring in here
```suggestion
            pages = _.map(hits, (hit) => ({ ...hit_source, uri: hit._id })),
```